Makefile

```
@@ -72,8 +72,8 @@ check-dependencies:
 .PHONY: check-dependencies
 
 ## start: Start all processes needed for the demo.
-start:
-	@docker compose up --detach
+start: stop
```

```suggestion
start: 
```

[nit] I removed this in a separate PR b/c it can be useful to invoke `make start` multiple times without shutting everything down.

For example if you comment out the celestia-prover in docker-compose run `make start` and then later want to run the celestia-prover in docker, you uncomment and run `make start` again.

This avoids having to stop and delete the whole stack.